src/cocotb_tools/makefiles/simulators/Makefile.vcs

```diff
@@ -58,9 +58,15 @@ ifdef VERILOG_INCLUDE_DIRS
 endif
 
 ifeq ($(PYTHON_ARCH),64bit)
-    EXTRA_ARGS += -full64
+    COMPILE_ARGS += -full64
 endif
 
+# link in libraries even if the linker thinks the library is not needed
+COMPILE_ARGS += -LDFLAGS -Wl,--no-as-needed
+
+# quiet mode
+COMPILE_ARGS += -q
```

Comment:
We generally set only the strictly required command-line options, since users can easily add command-line arguments, but it's hard to remove them.